include/fc/io/fstream.hpp

```diff
@@ -3,16 +3,19 @@
 #include <fc/filesystem.hpp>
 #include <fc/io/iostream.hpp>
 
+#include <fstream>
+
+using namespace std;
```

Comment:
Placing a using clause in a header is a very bad idea. Please remove this line.

Reply:
ok sure ! Thank you @jmjatlanta